charts/knative-serving/templates/configmaps/deployment.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-deployment
+  labels:
+    app.kubernetes.io/part-of: knative-serving
+    app.kubernetes.io/version: 1.0.1
+    serving.knative.dev/release: v1.0.1
+  {{- include "knative-serving.labels" . | nindent 4 }}
+  annotations:
+    knative.dev/example-checksum: dd7ee769
+data:
+  _example: {{ .Values.configDeployment.Example | quote }}
```

Comment:
The `_example` field is just for demonstration purposes as I can remember @tpiperatgod, right?. Maybe we needn't set it to a helm value.

Reply:
yes, it's just for demonstration

Reply:
Then we needn't to turn the _example into a value

Reply:
already move to template file